flink-runtime/src/test/java/org/apache/flink/runtime/net/SSLUtilsTest.java

```diff
@@ -386,7 +386,8 @@ public void testCreateSSLEngineFactory() throws Exception {
 		serverConfig.setString(SecurityOptions.SSL_ALGORITHMS, "TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA256");
 
 		final SSLHandlerFactory serverSSLHandlerFactory = SSLUtils.createInternalServerSSLEngineFactory(serverConfig);
-		final SslHandler sslHandler = serverSSLHandlerFactory.createNettySSLHandler();
+		// note: a 'null' allocator seems to work here (probably only because we do not use the ssl engine!)
+		final SslHandler sslHandler = serverSSLHandlerFactory.createNettySSLHandler(null);
```

Comment:
`io.netty.buffer.UnpooledByteBufAllocator#DEFAULT`?

Reply:
actually, a later commit in the PR already fixed that :)